protozfits/tests/test_File.py

```diff
@@ -0,0 +1,52 @@
+import pkg_resources
+import os
+from protozfits import File
+
+example_file_path = pkg_resources.resource_filename(
```

Comment:
Is there a way to run the test an all zfits files contained in tests/resources ?  At the moment it seems we are only doing tests on one file

Reply:
Forget what I said... This test is satisfying : https://github.com/cta-sst-1m/protozfitsreader/blob/121778fd96270ed7cddaf88e0027b7b2b39788a3/protozfits/tests/test_R1_example_file.py#L45